static/js/settings_org.js

```diff
@@ -107,6 +107,46 @@ exports.populate_auth_methods = function (auth_methods) {
     }
 };
 
+var realm_notif_stream;
+
+exports.update_notifications_stream = function (stream_id) {
```

Comment:
maybe this should be spelled `render_notifications_stream_ui`, for clarity?  This sounds like it might send something to the server.